api/core/evaluate.js

```diff
@@ -9,7 +9,7 @@ import {
 import { getApi } from './api.js';
 import { toEcmascript } from '@jsxcad/compiler';
 
-export const evaluate = async (ecmascript, { api, path }) => {
+export const evaluate = async (ecmascript, { api, id, path }) => {
```

Comment:
What is the ID used for?

Reply:
Ah, it tells us which definition we're evaluating.
e.g., for const foo = bar(); id === 'foo'.

Reply:
Ahhh that makes sense